### ISSUE TITLE
Error with Plugy stash:  Unexpected header ST for an item on page 39

### ISSUE THREAD

**Issue by zepcheese** [youdz/d2-stash-organizer#43]
Just came to organize my stash (from classic LOD plugy) and found this issue as a popup:

Unexpected header ST for an item on page 39

My pg39 has potions, keys, and a few Meph. Soulstone of varying levels.  That's it.
This stash page has worked in the past.

**Comment by youdz** [comment]
Yes, I've been getting a couple reports like that, I'll fix it this week-end. I'm sorry I can't find the time for it until then. 😞

**Comment by youdz** [comment]
This should be fixed now, I just pushed a fix for #42 which failed for the same reason, in their case a Khalim's Eye.

I'm closing this issue, but please reopen it if your save file still does not work.

**Comment by zepcheese** [comment]
no worries.  working fine for me now.  thanks!
